projects/arctic_lstm_speculator/scripts/llama3.1-70b.sh

```diff
@@ -0,0 +1,43 @@
+# Tested with 8xH100 node
+
+DATA_GEN=1
+TRAIN=1
+data_save_folder_name="llama31_70b_tmp"
+vllm_tensor_parallel=4
+script_save_path="llama31_70b_data_gen_scripts"
+total_num_of_scripts=8
+
+train_config_file="llama3.1-70b.yaml"
+# must be aligned with the config file
+model_name="meta-llama/Llama-3.1-70B-Instruct"
+data_concat_folder_name="llama31_70b_data"
+spec_drafter_name="spec-decode-llama-31-70b"
+num_speculative_tokens=3
+
+
+# data generation
+if [ "$DATA_GEN" -eq "1" ]; then
+  pip install vllm
+  rm -r $script_save_path ${script_save_path}_tmp
+  python mlp_speculator/data_generation/data_gen_script_maker.py --model_name=$model_name \
```

Comment:
```suggestion
  python speculator/data_generation/data_gen_script_maker.py --model_name=$model_name \
```

This should be `speculator` not `mlp_speculator`, correct?

Change needs to be propagated to other paths as well.